backend/app/main.py

```diff
@@ -53,3 +56,42 @@ async def db_is_alive(session: Session = Depends(get_session)):
         raise HTTPException(
             status_code=status.HTTP_503_SERVICE_UNAVAILABLE, detail="DB is unreachable"
         )
+
+
+@app.post("/create_document")
```

Comment:
I: Add HTTP response status code.
```suggestion
@app.post(
    "/create_document",
    status_code=status.HTTP_201_CREATED,
    response_model=DocumentPublic # Will need to create this model
)
```

And it sounds like we will go the UDF route to create the document, with the client hitting the DB when starting a new doc.